R/system.R

```diff
@@ -1482,3 +1482,15 @@ read_rds_file <- function(file, refhook = NULL){
     readRDS(file, refhook)
   }
 }
+
+#'Wrapper for readr::rea_lines to support vector to data frame conversion
+#'@export
+read_raw_lines <- function(file, locale = readr::default_locale(), na = character(),
+                            skip = 0, n_max = Inf, progress = interactive()){
+  loadNamespace("readr")
+    # if it's local file simply call readr::read_delim
+  x <- readr::read_lines(file, locale = locale, na = na, skip = skip, n_max = n_max, progress = progress)
+  df <- as.data.frame(x)
```

Comment:
"x" will be the column name.
column name with meaning like "line" might be better.